feat: modifiche di stesura del paragrafo dedicato agli ITS (#239)
Il-Libro-Open-Source/book: docs/it/università-vs-self-made.md

# Università VS Self-Made

## La terza via: l’ITS

Nel momento in cui si vuole intraprendere la via della “specializzazione” da developer, una volta concluse le scuole superiori, spesso ci si trova davanti a quello che è, a tutti gli effetti, un bivio: _intraprendere la via dell’Università o iniziare un percorso di autoformazione?_

> La risposta non è sempre così scontata, vista anche la soggettività del tema. Nel primo caso stiamo parlando di un percorso che dura minimo tre anni (o cinque, a seconda dell’obiettivo), strutturato e tenuto da docenti formati e che seguono un piano di studi ben preciso, mentre nel secondo abbiamo una durata potenzialmente ignota e che si basa esclusivamente sulla nostra organizzazione, disciplina e motivazione.

Reviewer: ```suggestion
La risposta non è sempre così scontata, vista anche la soggettività della scelta. Nel primo caso stiamo parlando di un percorso che dura minimo tre anni (o cinque, a seconda dell’obiettivo), strutturato e tenuto da docenti formati e che seguono un piano di studi ben preciso, mentre nel secondo abbiamo una durata potenzialmente ignota e che si basa esclusivamente sulla nostra organizzazione, disciplina e motivazione.
```

> Molte persone però non sono a conoscenza di una terza via, ovvero degli [ITS (_Istituto Tecnico Superiore_)](https://www.miur.gov.it/web/guest/tematica-its) di programmazione, degli enti solitamente finanziati dalla regione e/o da alcune aziende del settore, il cui scopo è quello di formare nel giro di un paio d’anni delle figure professionali da inserire nel mondo dell’IT.

Reviewer: ```suggestion
Molte persone spesso però sottovalutano o addirittura non sono a conoscenza di una terza via, ovvero degli [ITS (_Istituto Tecnico Superiore_)](https://www.miur.gov.it/web/guest/tematica-its) di programmazione, degli enti solitamente finanziati dalla regione e/o da alcune aziende del settore, il cui scopo è quello di formare nel giro di un paio d’anni delle figure professionali da inserire nel mondo dell’IT.
```

### Un ibrido tra Teoria e Pratica

**_“Com'è strutturato il percorso formativo di un ITS?”_**

> Per integrare in modo corretto e graduale la “Teoria” e la “Pratica”, il piano formativo standard di un ITS prevede:

Reviewer: ```suggestion
Per integrare in modo corretto e graduale la “Teoria” e la “Pratica”, il piano formativo standard di un ITS generalmente prevede:
```

> - **Primo Anno**: Prevalentemente dedicato alle **lezioni in aula**, dove si ha l'opportunità di apprendere dalle competenze di **docenti aziendali**<sup>1</sup> e universitari, i concetti, i linguaggi e gli strumenti fondamentali della figura professionale a cui è dedicato il corso.

Reviewer: ```suggestion
- **Primo Anno**: Prevalentemente dedicato alle **lezioni in aula**, dove si ha l'opportunità di apprendere dalle competenze di **docenti aziendali**<sup>1</sup> e universitari, i concetti, i linguaggi e gli strumenti fondamentali della figura professionale che si ambisce a diventare.
```

  Inoltre c’è da menzionare che in alcuni ITS viene offerta la possibilità di prepararsi e sostenere esami per certificazioni esterne quali: [CISCO IT Essentials](https://www.netacad.com/courses/os-it/it-essentials), [ECDL](https://www.aicanet.it/aica/qualita/ecdl-full-standard) e [PET/FIRST/ecc.](https://www.cambridgeenglish.org/exams-and-tests/) per citarne solo alcune.

> - **Secondo Anno**: Dopo una breve fase di lezioni dedicate agli ultimi moduli, mirate a esplorare concetti più avanzati, come l'_Architettura del Software_ per esempio, l'attenzione si sposta principalmente sullo **stage lavorativo**, nonché ultima attività prima dell’esame finale (necessario per conseguire un _Diploma di [V livello EQF](https://europa.eu/europass/it/description-eight-eqf-levels)_).

Reviewer: Quali sono questi moduli? Trovo questo paragrafo poco chiaro

Author: Ciao, non sono entrato troppo nel merito in quanto questi moduli possono variare da corso a corso.
Ho citato l'architettura software siccome è stato uno di questi moduli "avanzati" che ho affrontato personalmente nel mio corso; ma come detto, è un esempio.
Se sei d'accordo posso cercare di rendere il tutto un po' più chiaro citando questo aspetto sulla differenza da corso a corso

Reviewer: Secondo me non è sbagliato dire "Io ho fatto questo e questi sono i moduli - che ogni ITS struttura a modo suo"

Reviewer 2: L'aggiunta della parola "generalmente" in alto per me è sufficiente per indicare che quello che si mostra è un esempio e non l'unica via. Che ne dici @AngeloAvv ?

Reviewer: Va bene!

**_“Come posso trovare gli ITS?”_**

Prima di rispondere a questa domanda, è bene precisare che gli ITS in Italia offrono una vasta gamma di corsi dedicati a settori diversi, tra cui tecnologia, industria, e servizi. Per ovvi motivi in questo libro, più precisamente in questo capitolo, si parlerà di quelli legati al settore IT.

> Chiusa questa parentesi, il sito dell’**_[INDIRE](https://www.indire.it/progetto/its-istituti-tecnici-superiori/dove-sono-gli-its/)_** (_Istituto Nazionale Documentazione Innovazione Ricerca Educativa_) è sicuramente la miglior risposta a questa domanda. Ovviamente non l’unica, ma sicuramente tra tutte le pagine risulta essere la più completa, presentando anche a sostegno i dati relativi agli ITS della regione e all’occupazione degli studenti diplomati.

Reviewer: ```suggestion
Chiusa questa parentesi, il sito dell’**_[INDIRE](https://www.indire.it/progetto/its-istituti-tecnici-superiori/dove-sono-gli-its/)_** (_Istituto Nazionale Documentazione Innovazione Ricerca Educativa_) è sicuramente la migliore risorsa per ottenere una risposta a questa domanda. Ovviamente non l’unica, ma sicuramente tra tutte le pagine risulta essere la più completa, presentando anche a sostegno i dati relativi agli ITS della regione e all’occupazione degli studenti diplomati.
```

**“_Perché l’ITS potrebbe risultare un’ottima via?”_**

Innanzitutto come già citato è un corso molto breve (2 anni), a numero chiuso (quindi con un test di ammissione) e alla portata di tutti dal punto di vista economico, dal momento che l’iscrizione è gratuita o a costo molto ridotto. Solitamente viene fornito un PC da poter utilizzare durante le lezioni e dedicato completamente alla programmazione, ma soprattutto, ciò che contraddistingue l’ITS (e il motivo per cui viene citato in questo capitolo) è la sua **dualità tra lezioni in aula e stage in azienda** per poter testare in modo pratico “sul campo” i concetti appresi.

> È vero, la preparazione concettuale/teorica potrebbe risultare meno solida di quella universitaria, ma sicuramente favorevole in ambito aziendale, in quanto più improntata al mondo del lavoro. Ed è proprio qua, che sfruttando le occasioni lavorative e gli incontri con i professionisti del settore deve subentrare la proattività e la curiosità, per sfruttare al meglio le competenze di partenza che abbiamo acquisito!

Reviewer: Puoi rivedere la seconda parte del paragrafo? La trovo poco chiara  poco scorrevole

> È vero, la preparazione concettuale e teorica potrebbe risultare meno solida rispetto a quella universitaria, ma è sicuramente più vantaggiosa in ambito aziendale, essendo maggiormente orientata al mondo del lavoro. Proprio in questo contesto, sfruttando le opportunità lavorative e gli incontri con i professionisti del settore, dobbiamo far emergere la nostra proattività e curiosità per valorizzare al meglio le competenze di base che abbiamo acquisito!

Reviewer 3: ```suggestion
Qualcuno potrebbe obiettare dicendo che la preparazione concettuale e teorica potrebbe risultare meno solida rispetto a quella universitaria, anche se questa scelta, come spiegato in precedenza, predispone molte attività in ambito aziendale, essendo maggiormente orientata al mondo del lavoro, e quindi aiuta nel "salto". Proprio in questo contesto, sfruttando le opportunità lavorative e gli incontri con i professionisti del settore, dobbiamo far emergere la nostra proattività e curiosità per valorizzare al meglio le competenze di base che abbiamo acquisito!
```

> Per esempio, le lezioni di Python che ho seguito in Accademia mi hanno incuriosito e vorrei saperne ancora di più su questo linguaggio. Ottimo! Partendo da questa "infarinatura" posso apprendere più facilmente, anche in maniera autonoma.

Reviewer: ```suggestion
Per esempio, le lezioni di Python che ho seguito in Accademia hanno solleticato la mia curiosità e adesso non vedo l'ora di affinare le mie conoscenze su questo linguaggio. Partendo da questa "infarinatura" posso apprendere più facilmente, anche in maniera autonoma.
```

> Per esempio, le lezioni di Python che ho seguito in Accademia hanno solleticato la mia curiosità e adesso non vedo l'ora di affinare le mie conoscenze su questo linguaggio. Partendo da questa "infarinatura" posso apprendere più facilmente, anche in maniera autonoma.

Reviewer 3: Forse mi sarei tenuta sul generico, senza entrare troppo nel "soggettivo".

```suggestion
Per esempio, è possibile fruire di lezioni di Python...
```

Author: Capisco l'osservazione, che ci sta, e motivo il perché ho scelto questa forma. Come per il discorso dei "moduli" che variano da corso a corso, anche in questo caso ho voluto fare un esempio pratico, citando in stile esperienza personale (come se lo stessi raccontando al lettore), un linguaggio che non è detto venga effettivamente trattato. Ovvio che se preferiamo tenere il tutto un po' più "oggettivo" posso tranquillamente rivedere la forma.

> Tramite questo percorso, passando per gli stage, si possono acquisire anche competenze su argomenti che non sono stati affrontati in aula. Magari potrei trovarmi a svolgere uno stage in un’azienda che usa VUE.js. Ho seguito le lezioni su JavaScript, ma questo framework non è mai stato trattato. Ecco un’ottima opportunità per impararlo ora.

Reviewer: ```suggestion
Tramite questo percorso, passando per gli stage, si possono acquisire anche competenze su argomenti che non sono stati affrontati in aula. Ad esempio, in futuro potrei trovarmi a svolgere uno stage in un’azienda che usa un framework specifico: avendo seguito le lezioni su JavaScript, ho acquisito delle solide basi che non potranno far altro che semplificare il processo di apprendimento di una nuova tecnologia basta su questo linguaggio di programmazione.
```

> Tramite questo percorso, passando per gli stage, si possono acquisire anche competenze su argomenti che non sono stati affrontati in aula. Ad esempio, in futuro potrei trovarmi a svolgere uno stage in un’azienda che usa un framework specifico: avendo seguito le lezioni su JavaScript, ho acquisito delle solide basi che non potranno far altro che semplificare il processo di apprendimento di una nuova tecnologia basta su questo linguaggio di programmazione.

Reviewer 3: Come sopra. Rimuoverei la parte soggettiva, e illustrerei i vantaggi da un punto di vista più fattuale (perché, oggettivamente, ci sono!)

```suggestion
Attraverso gli stage che si svolgono seguendo questo percorso, si possono acquisire anche competenze su argomenti che non sono stati affrontati in aula. Ad esempio, in futuro ...```

Author: Anche in questo caso la motivazione per cui ho scelto questa forma è quella menzionata nella review precedente. Ovviamente se modifico uno di questi due paragrafi per renderli più "oggettivi", andrò ad agire anche sull'altro per tenerli coerenti

> # Università VS Self-Made

Reviewer 2: ```suggestion
---
layout: default
title: Università VS Self-Made
nav_order: 20
---

<!-- prettier-ignore-start -->
# Università VS Self-Made
{: .no_toc }

- TOC
{:toc}

<!-- prettier-ignore-end -->
```

# Università VS Self-Made

## La terza via: l’ITS

> Nel momento in cui si vuole intraprendere la via della “specializzazione” da developer, una volta concluse le scuole superiori, spesso ci si trova davanti a quello che è un bivio: _intraprendere la via dell’Università o iniziare un percorso di autoformazione?_

Reviewer 4: ```suggestion
Nel momento in cui si vuole intraprendere la via della “specializzazione” da developer, una volta concluse le scuole superiori, spesso ci si trova davanti a quello che è, a tutti gli effetti, un bivio: _intraprendere la via dell’Università o iniziare un percorso di autoformazione?_
```

oppure, semplicemente `spesso ci si trova davanti a un bivio:`

Diciamo che mettere "a quello che è" te lo chiama proprio il "a tutti gli effetti" imho

### Un ibrido tra Teoria e Pratica

**_“Com'è strutturato il percorso formativo di un ITS?”_**

Per integrare in modo corretto e graduale la “Teoria” e la “Pratica”, il piano formativo standard di un ITS prevede:

- **Primo Anno**: Prevalentemente dedicato alle **lezioni in aula**, dove si ha l'opportunità di apprendere dalle competenze di **docenti aziendali**<sup>1</sup> e universitari, i concetti, i linguaggi e gli strumenti fondamentali della figura professionale a cui è dedicato il corso.
  Inoltre c’è da menzionare che in alcuni ITS viene offerta la possibilità di prepararsi e sostenere esami per certificazioni esterne quali: [CISCO IT Essentials](https://www.netacad.com/courses/os-it/it-essentials), [ECDL](https://www.aicanet.it/aica/qualita/ecdl-full-standard) e [PET/FIRST/ecc.](https://www.cambridgeenglish.org/exams-and-tests/) per citarne solo alcune.

> <br>

Reviewer 4: Sinceramente toglierei i `br` e manterrei la formattazione standard dell'elenco puntato.
Ad avvalorare il mio pensiero porto una maggiore coerenza tra i capitoli passasti e futuri.

Lo stacco dei prossimi paragrafetti, allo stesso modo, lo farei evitando il br

Chiusa questa parentesi, il sito dell’**_[INDIRE](https://www.indire.it/progetto/its-istituti-tecnici-superiori/dove-sono-gli-its/)_** (_Istituto Nazionale Documentazione Innovazione Ricerca Educativa_) è sicuramente la miglior risposta a questa domanda. Ovviamente non l’unica, ma sicuramente tra tutte le pagine risulta essere la più completa, presentando anche a sostegno i dati relativi agli ITS della regione e all’occupazione degli studenti diplomati.

<br>

**“_Perché l’ITS potrebbe risultare un’ottima via?”_**

Innanzitutto come già citato è un corso molto breve (2 anni), a numero chiuso (quindi con un test di ammissione) e alla portata di tutti dal punto di vista economico, dal momento che l’iscrizione è gratuita o a costo molto ridotto. Solitamente viene fornito un PC da poter utilizzare durante le lezioni e dedicato completamente alla programmazione, ma soprattutto, ciò che contraddistingue l’ITS (e il motivo per cui viene citato in questo capitolo) è la sua **dualità tra lezioni in aula e stage in azienda** per poter testare in modo pratico “sul campo” i concetti appresi.

> È vero, la preparazione concettuale/teorica potrebbe risultare meno solida di quella universitaria, ma sicuramente giocherà a nostro favore in ambito aziendale, in quanto più improntata al mondo del lavoro. Ed è proprio qua, che sfruttando le occasioni lavorative e gli incontri con i professionisti del settore deve subentrare la nostra proattività e la nostra curiosità, per sfruttare al meglio le competenze di partenza che abbiamo acquisito!

Reviewer 4: ```suggestion
È vero, la preparazione concettuale/teorica potrebbe risultare meno solida di quella universitaria, ma sicuramente favorevole in ambito aziendale, in quanto più improntata al mondo del lavoro. Ed è proprio qua, che sfruttando le occasioni lavorative e gli incontri con i professionisti del settore deve subentrare la proattività e la curiosità, per sfruttare al meglio le competenze di partenza che abbiamo acquisito!
```

Mi sembra sia sempre stata usata la terza plurale fino adesso, quindi la prima qui mi sembra inadeguata

Per esempio, le lezioni di Python che ho seguito in Accademia mi hanno incuriosito e vorrei saperne ancora di più su questo linguaggio. Ottimo! Partendo da questa "infarinatura" posso apprendere più facilmente, anche in maniera autonoma.

Tramite questo percorso, passando per gli stage, si possono acquisire anche competenze su argomenti che non sono stati affrontati in aula. Magari potrei trovarmi a svolgere uno stage in un’azienda che usa VUE.js. Ho seguito le lezioni su JavaScript, ma questo framework non è mai stato trattato. Ecco un’ottima opportunità per impararlo ora.

> L'importante è alimentare sempre questo “fuoco”, questa sete di conoscenza, che oltre ad essere fondamentale in questo settore (perché non si smette mai di imparare nel mondo IT 🤓) ci aiuterà a crescere come figura professionale e a farci notare da aziende dove è più facile fare esperienza e assimilare concetti nuovi.

Reviewer 4: ```suggestion
L'importante è alimentare sempre questo “fuoco”, questa sete di conoscenza che, oltre ad essere fondamentale in questo settore (perché non si smette mai di imparare nel mondo IT 🤓), ci aiuterà a crescere come figura professionale e a farci notare da aziende dove è più facile fare esperienza e assimilare concetti nuovi.
```
